communication/src/chunked_transfer.cpp

```diff
@@ -33,6 +33,18 @@ ProtocolError ChunkedTransfer::handle_update_begin(
     if (actual_len >= 20 && queue[7] == 0xFF)
     {
         flags = decode_uint8(queue + 8);
+
+#if PLATFORM_ID == 10
```

Comment:
Could we possibly do this in a more generic way? Bringing platform-specific stuff into communication library doesn't sound like a good idea.

One possible approach I can think of from the top of my head is adding a new protocol command (`ProtocolCommands` / `spark_protocol_command`) that would allow to force `ChunkedTransfer` instance to use non-Fast-OTA mode. Having such a command available we could call it either from cellular_hal, once device type is detected, or at least from the `system`.

Reply:
This is only a temporary solution while we spend some more time troubleshooting Fast OTA on LTE.  We could also evolve to a more generic strategy if we determine Slow OTA has to stay.

Reply:
I ended up moving this code to system_task.cpp and pushing the FAST_OTA override into the communications library as we discussed.